internal/validations/observability/outputs/validate_secrets_test.go

```diff
@@ -40,5 +40,15 @@ var _ = Describe("validating output secrets", func() {
 			conds := ValidateSecretsAndConfigMaps(spec, secrets, configMaps)
 			Expect(conds).To(Not(BeEmpty()))
 		})
+		It("should validate secrets if TLS nil", func() {
+			spec.TLS = nil
+			conds := ValidateSecretsAndConfigMaps(spec, secrets, configMaps)
+			Expect(conds).To(Not(BeEmpty()))
```

Comment:
Why should this fail if TLS is empty?  It is a valid for an output to not have TLS settings

Reply:
code fail [here](https://github.com/openshift/cluster-logging-operator/blob/master/internal/validations/observability/outputs/validate.go#L23) NPE if TLS is nil

Reply:
> code fail here NPE if TLS is nil

Shouldnt the fix then be to fix the nil pointer in :arrow_up: ? It is acceptable to not provide TLS spec.  This just means you don't need these tls configuration